src/main.rs

```diff
@@ -82,9 +82,7 @@ impl Run {
         .context("failed to compose Spin app, test, and virtualized Spin environment")?;
 
         let tests = run_tests(&test_name, test_target, encoded, manifest)?;
-        let _ = libtest_mimic::run(&libtest_mimic::Arguments::default(), tests);
-
-        Ok(())
+        libtest_mimic::run(&libtest_mimic::Arguments::default(), tests).exit();
```

Comment:
Non blocking, good for future follow up: we probably want a test to validate this behavior

Reply:
Good call. Unfortunately, we don't have any tests right now which is tracked in #42